compiler_opt/es/blackbox_test_utils.py

```diff
@@ -33,14 +33,25 @@ def __init__(self, *, delta=1.0, initial_value=0.0):
     self._delta = delta
 
   def compile(self, policy: policy_saver.Policy,
```

Comment:
It looks like these changes aren't relevant anymore?

Reply:
Compile method use LoadedModuleSpec with the recent changes anyway. So left it as such.